tests/PS01/Dataset/README.md

```diff
@@ -0,0 +1,94 @@
+# Test dataset
+
+| Test code | Test author     | Test dataset source | Test direction |
+|-----------|-----------------|---------------------|----------------|
+| PS01      | Stefan Jaud     | SBB                 | Export         |
+
+## Content
+- [Test dataset](#test-dataset)
+  - [Content](#content)
+  - [Model Dataset](#model-dataset)
+  - [Geographic Coordinate System properties](#geographic-coordinate-system-properties)
+
+
+## Model Dataset
+
+This dataset is an example of what the project setup model could contain.
+
+The project defines default units:
+
+- *metre* as `LENGTHUNIT`
+- *square metre* as `AREAUNIT`
+- *cubic metre* as `VOLUMEUNIT`
+- *degree* as `PLANEANGLEUNIT`
+- *kilogram* as `MASSUNIT`
+- *second* as `TIMEUNIT`
+- *EUR* as monetary unit
```

Comment:
We are in Switzerland :)

Reply:
I hoped you'd pick this one up :) Will amend accordingly.

Reply:
Changed to `CHF` in 797d5d12321585f5311e7ba8e3db63b410a0b474